src/IKEA/Tradfri/Helper/Runner.php

```diff
@@ -12,19 +12,25 @@
 class Runner
```

Comment:
My gateway can only handle a certain amount of concurrent requests, I would suggest to make a queue and use promises to return the data.

Reply:
hi, I added your suggestion and will add a command queue soon. I have plans to change the command handling anyway. WIth an Invoker and a Receiver like https://github.com/domnikl/DesignPatternsPHP/tree/master/Behavioral/Command